Routes/Question/Qsts.js

```diff
@@ -1,53 +1,64 @@
 //Alexis
-
 var Express = require('express');
 var Tags = require('../Validator.js').Tags;
 var router = Express.Router({caseSensitive: true});
 var async = require('async');
 
-router.baseURL = '/Cnvs';
+router.baseURL = '/Qsts';
+
 
+// return questions with owner specified or current AU
 router.get('/', function(req, res) {
-   if (req.query.hasOwnProperty("owner")) {
-      req.cnn.chkQry('select id, ownerId, title, UNIX_TIMESTAMP(lastMessage)' +
-       ' * 1000 as lastMessage from Conversation where ownerId = ?', 
-       parseInt(req.query.owner),
-       function(err, cnvs) {
-          if (!err)
-             res.json(cnvs);
-          
-          req.cnn.release();
-       });
+   var cnn = req.cnn;
+   var ownerId = req.query.owner || req.session.id;
+
+   if (ownerId !== req.session.id) {
+      cnn.chkQry('select q.id, ownerId, c.title as category, q.title from' +
+       ' Question as q ' +
+       'inner join Category as c on q.categoryId = c.id where ownerId = ?',
+       ownerId,
+      function(err, qsts) {
+         if(!err) {
+            res.json(qsts);
+         }
+         cnn.release();
+      });
    }
    else {
-      req.cnn.chkQry('select id, ownerId, title, UNIX_TIMESTAMP(lastMessage)' +
-       ' * 1000 as lastMessage from Conversation', null,
-       function(err, cnvs) {
-          if (!err)
-             res.json(cnvs);
-
-          req.cnn.release();
+      cnn.chkQry('select q.id, ownerId, c.title as category, q.title,' +
+       ' q.answer from Question as q ' +
+       'inner join Category as c on q.categoryId = c.id where ownerId = ?',
+      ownerId,
```

Comment:
Should be an array for the chkQry parameters I believe. Aka [ownerId] instead of ownerId.